.github/workflows/build-and-test.yml

```diff
@@ -38,6 +39,8 @@ jobs:
           container: ornladios/adios2:ci-suse-pgi
         - jobname: suse-pgi-openmpi
           container: ornladios/adios2:ci-suse-pgi-openmpi
+        - jobname: debian-sid-openmpi
+          container: debian:sid
 
     steps:
     - uses: actions/checkout@v2
```

Comment:
It looks like we need to move the checkout step to after the `Setup` step since the action isn't able to find an installed git in the debian build so the dashboard submissions aren't showing up.

Reply:
Can you run the checkout after the setup? Doesn't the script need to be there (checked out) in order to run it? I'll try it and see what happens.

Reply:
Yeah, that failed miserably. Not surprising.

I guess I'll have to make a minimal image that has `git` initially installed just to bootstrap the process. The rest of the dependencies will still be updated on the fly.

Reply:
I think using the upstream image like you have now is good so maybe you can add an initial first step to just install git and make it conditional on not using an ornladios image and matches "debian"

Reply:
Good point. I'll try that.

Reply:
OK, looks like that worked.